samples/demos/irb/README.md

```diff
@@ -1,10 +1,47 @@
-# IRB approval sample
+# Institutional Review Board (IRB) Sample
 
-Get redis
+This demo implements an FPC-based IRB experiment approval services to protect the confidentiality of data being used in analytical experiments. 
```

Comment:
I would add something like that this applies for clinical trials in the farmaceutical industry ...